### ISSUE TITLE
Max time limit

### ISSUE THREAD

**Issue by Gr-Nobel** [tetsuwo/website-blocker-chrome.ext#126]
is there a way to set a site's max time limit.
like blocking facebook after 10 minutes of use.

**Comment by rotten77** [comment]
No, there is no way to set this but I wouldn't expect such a feature from this extension. I am using StayFocusd for the spent-time/usage limit.
